base/applications/taskmgr/about.c

```diff
@@ -1,9 +1,9 @@
 /*
- *  ReactOS Task Manager
+ * ReactOS Task Manager
```

Comment:
Since you're fixing headers here, maybe it's worth to just switch to our new header syntax. Like this:
```c
/*
 * PROJECT:     ReactOS Task Manager
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     ...
 * COPYRIGHT:   ...
 */
```
See https://reactos.org/wiki/Coding_Style

Reply:
Good idea, I will do that at some later point.